src/redux.js

```diff
@@ -176,6 +176,7 @@ export function setInstructionsKeyCallback(instructionsKeyCallback) {
 }
 
 export function setCurrentPanel(currentPanel) {
+  reportPanelView(currentPanel);
```

Comment:
I think the idiomatic Redux approach would be to do the report in the `SET_CURRENT_PANEL` handler below?